src/openxr/OpenXRApi.cpp

```diff
@@ -833,29 +793,27 @@ bool OpenXRApi::initialiseInstance() {
 	return true;
 }
 
-bool OpenXRApi::initialiseExtensions() {
+bool OpenXRApi::initialise_extensions() {
 	XrResult result;
```

Comment:
Note, I'm slowly renaming things according to our naming convention discussion. I'll probably do a PR soon that goes through most of the current code base but for now I'm only concentrating on things impacted by this PR.